### ISSUE TITLE
translation issues

### ISSUE THREAD

**Issue by redouane** [flask-admin/flask-admin#32]
flask-admin uses babel domains, which are not yet on the official flask-babel repo ( i think mrjoes sent a pull request 3 months ago)
as a result of that, flask-admin currently cannot be translated, unless you fork, or clone/use mr joes flask-babel fork or wait for flask-babel to pull mrjoes commits.

this dependency / issue isnt documented anywhere, and flask-admin does not complain about the them not being implemented, and instead defaults to the currently set strings.

tl;dr : why implement it that way, ( dependency on not yet pulled to official flask-babel repo ) ? 

thanks.
